front-end/src/MyShows.js

```diff
@@ -58,7 +56,7 @@ const ShowGrid = (props) => {
 }
 
 // For now, a very simple + non-function search.
```

Comment:
should read "non-functional"

Reply:
Looks good otherwise!

Reply:
Fixed, thanks Lev! Do you think I could merge it in now?